tests/tests/database_api_tests.cpp

```diff
@@ -692,4 +692,101 @@ BOOST_AUTO_TEST_CASE( lookup_vote_ids )
 
 } FC_LOG_AND_RETHROW() }
 
+BOOST_AUTO_TEST_CASE(get_account_limit_orders)
+{ try {
+
+   ACTORS((seller));
+
+   const auto& bitcny = create_bitasset("CNY");
+   const auto& core   = asset_id_type()(db);
+
+   int64_t init_balance(10000000);
+   transfer(committee_account, seller_id, asset(init_balance));
+   BOOST_CHECK_EQUAL( 10000000, get_balance(seller, core) );
+
+   /// Create 250 versatile orders
+   for (int i = 0 ; i < 50 ; ++i) {
+     BOOST_CHECK(create_sell_order(seller, core.amount(100), bitcny.amount(250)));
+   }
+
+   for (int i = 1 ; i < 101 ; ++i) {
+     BOOST_CHECK(create_sell_order(seller, core.amount(100), bitcny.amount(250 + i)));
+     BOOST_CHECK(create_sell_order(seller, core.amount(100), bitcny.amount(250 - i)));
+   }
+
+   graphene::app::database_api db_api(db);
+   std::vector<limit_order_object> results;
+   limit_order_object o;
+
+   // query with no constraint, expected:
+   // 1. up to 101 orders returned
+   // 2. orders were sorted by price desendingly
+   results = db_api.get_account_limit_orders(seller.name, "BTS", "CNY");
+   BOOST_CHECK(results.size() == 101);
+   for (int i = 0 ; i < results.size() - 1 ; ++i) {
+     BOOST_CHECK(results[i].sell_price >= results[i+1].sell_price);
+   }
```

Comment:
Check if the first item and the last item are correct (also best to add this check to test cases below).